src/rgw/rgw_rados.cc

```diff
@@ -6420,15 +6407,28 @@ std::string get_obj_data::d3n_deterministic_hash(std::string oid)
   std::string location = g_conf()->rgw_d3n_l2_datacache_hosts;
   string delimiters(",");
 
-  std::vector<std::string> tokens = split(location, ",");
+  std::vector<std::string> tokens;
+  boost::split(tokens, location, boost::is_any_of(",")); 
```

Comment:
I'm sure this is fine, but doesn't @adamemerson have some accelerated "spliterator" thing?

Reply:
moved to the L2 PR, will address it there